src/System.IO.Pipelines.Text.Primitives/ReadableBufferExtensions.cs

```diff
@@ -109,14 +109,14 @@ public unsafe static ulong GetUInt64(this ReadableBuffer buffer)
                 ArraySegment<byte> data;
                 if (buffer.First.TryGetPointer(out pointer))
```

Comment:
Has it been tried to call buffer.First.Span and the primitive parsing routines that take a Span? I'd be curious to see how much difference there was in performance.